src/cozmo/robot.py

```diff
@@ -174,7 +174,7 @@ def _repr_values(self):
 
     def _encode(self):
         return _clad_to_engine_iface.RollObject(objectID=self.obj.object_id,
-                                                approachAngle_rad = 0, #I've chosen not to expose the approach angle, as it doesn't seem to have any effect
+                                                approachAngle_rad = 0, #Approach angle is not used, as it doesn't seem to have any effect
```

Comment:
Ah, figured it out - it looks like i need to set usePreDockPose=True in order for useApproachAngle=True to work.  I'll rework this to expose more configurability.